lab-cameron/routes/pokemon-routes.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
This module looks good, but for a first glance, might be good to space things out between your routes, i.e. indent a line between ``router.get`` and ``router.post``.